s3/replication/common/src/s3replicationcommon/jobs.py

```diff
@@ -23,24 +23,40 @@
 
 
 class Jobs:
-    def dumps(obj):
-        """json """
-        return json.dumps(obj._jobs, cls=JobJsonEncoder)
+    def to_json(self):
```

Comment:
change or ordering definition seems not required.

Reply:
done.